styles/find-and-replace.less

```diff
@@ -228,7 +228,20 @@ atom-workspace.find-visible {
   display: flex;
   flex-direction: column;
 
-  .panel-heading {
+  .preview-header {
```

Comment:
Not sure about the gradient and the top border, maybe that should be left for themes to add. Here where they would be removed:

```less
.preview-header {
    display: flex;
    padding: @component-padding/2 @component-padding/2 @component-padding/2 @component-padding;
    align-items: center;
    justify-content: space-between;
    font-weight: normal;
    border-bottom: 1px solid @panel-heading-border-color;
    background-color: @panel-heading-background-color;
  }
```

![screen shot 2016-08-24 at 3 42 28 pm](https://cloud.githubusercontent.com/assets/378023/17920796/6f56ec12-6a11-11e6-8119-41afb9e8e9ea.png)